source/opcode.cpp

```diff
@@ -236,10 +236,10 @@ int32_t spvOpcodeIsPointer(const SpvOp opcode) {
   switch (opcode) {
```

Comment:
spvOpcodeIsLogicalPointer maybe? or spvOpcodeReturnsLogicalPointer?
Not sure about the pulling into idUsage part though. It makes sense in that it could directly use the addressingModel variable, but on the other hand you still want a global function to test this (like the others in opcode.h), even if it isn't used anywhere else right now. An alternative could be to add an SpvAddressingModel argument to it (if the spvOpcodeIsPointer name is kept, otherwise it would be confusing).